consensus/hotstuff/votecollector/combined_vote_processor_v2.go

```diff
@@ -114,6 +114,25 @@ type CombinedVoteProcessorV2 struct {
 
 var _ hotstuff.VoteProcessor = (*CombinedVoteProcessorV2)(nil)
 
+func NewCombinedVoteProcessor(log zerolog.Logger,
```

Comment:
hotstuff integration tests would like to be able to create combined vote processor to inject mocked aggregator and reconstructor for simplified sig handling

Reply:
In that case, I would suggest to move this function to test code. 

No module should directly create combined vote processor, they should create through the factory. Having this function would make it a bit confusing, as there will be two different ways to create this instance.